user-guide/Advanced_Functionality/Databases/Indexing_database/Elasticsearch/dedicated_clustered_storage/Configuring_database/Configuring_multiple_Elasticsearch_clusters.md

```diff
@@ -21,6 +21,41 @@ Configuring multiple Elasticsearch clusters becomes crucial when on-premises set
 
 To configure this setup:
 
+## [From DataMiner 10.3.10/10.4.0 onwards](#tab/tabid-1)
+
+1. Got to *Apps > System Center > Database*.
+
+1. Database per cluster should already be selected. If this is not the case see [xref:Configuring database settings in cube](xref:Configuring_the_database_settings_in_Cube) to set up the basic Cassandra Cluster and Elastic configurations.
+
+1. Enable Multi cluster offload
+   
+1. Press **Add** to add an empty Elastic offload cluster to the list. You can offload to as many clusters as needed but only the main Elastic config above this list will be used to read from.
```

Comment:
Not exactly sure how to word this properly, but the amount of configs is unlimited but only the main (pre-existing) Elastic config is used as read database.